src/shared/molecules/MyDataTable/MyDataTable.tsx

```diff
@@ -244,12 +246,15 @@ const MyDataTable: React.FC<TProps> = ({
           return (
             <div>
               organization / project
-              <Sorter
-                name="name"
-                order={orderDirection}
-                onSortAscend={() => updateSort(['_project'])}
-                onSortDescend={() => updateSort(['-_project'])}
-              />
+              {query ||
```

Comment:
Is this intentional? Wouldn't the remainder of this expression not evaluate if `query` is truthy? 

I think it's also hard to understand the intended logic here. Are we saying that if `query` is falsy only then show the sorter? If so, I think using the following pattern here might be clearer: `query ? (render something) : ( render something else)`

Reply:
I see. I still think that it might be easier to understand the following syntax more than the current one. Not a must have though, so I'll approve the PR.

{
      query.trim().length === 0 
           ? (<Sorter
                    name="name"
                    order={orderDirection}
                    onSortAscend={() => updateSort(['_project'])}
                    onSortDescend={() => updateSort(['-_project'])}
                  />)
           : query
}